test/installers/msi_windows_test.go

```diff
@@ -0,0 +1,115 @@
+package installers_test
```

Comment:
This should be under `test/integration`

Reply:
The suffix must be `_int_windows_test.go` for the implicit windows build tag to be valid. Should I use an explicit build tag with `_int_test.go` suffix, or should I use the implicit `_int_windows_test.go` suffix?